docs/csharp/tour-of-csharp/tutorials/hello-world.yml

```
@@ -57,10 +57,11 @@ items:
     prints out the name.
 
     You can assign different values to any variable you declare. You can change
-    the name to one of your friends. Add these two lines in the interactive window
+    the name to one of your friends. Add these three lines in the interactive window
```

Let's make this more clear.  (GitHub won't let me apply a suggestion to all three lines, so we may need to edit by hand in addition to applying this suggestion):

```suggestion
    the name to one of your friends. Add these two lines in the interactive window
    following the code you've already added.  Make sure you keep the declaration
    of the `aFriend` variable and its initial assignment.
```